uwp/Pdf-Viewer/Concepts-and-Features/Working-with-custom-stamp-annotation.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: post
+title: Custom stamp annotation in UWP PDF Viewer control | Syncfusion
+description: Learn here all about Custom stamp annotation support in Syncfusion UWP PDF Viewer (SfPdfViewer) control and more.
+platform: UWP
+control: SfPdfViewerControl
+documentation: ug
+---
+
+# Working with custom stamp annotations in UWP PDF Viewer (SfPdfViewer)
+
+PDFViewer allows you to include any form of UIElement (Button, Entry, Label, Image, etc.) anywhere in the pages of the PDF Viewer as a custom stamp annotation. You can perform operations like add, move, resize, and delete the custom stamp annotations. Also, you can save and load the existing custom stamp annotation that is associated with the PDF document.
+
+N> Stamp annotations can be saved only by using the SaveAsync method.
+
+## Add a custom stamp
+
+The custom stamps can be added using the `AddStamp` methods.
+
+{% tabs %}
+{% highlight c# %}
+
```

Comment:
Please check whether we need to use C# code here

<img width="355" alt="image" src="https://github.com/syncfusion-content/uwp-docs/assets/65169077/fee5e64a-e832-4c68-a817-879ad34ce735">

Reply:
Used just to showcase the types of parameters. Xamarin Forms UG also showcases the same with C# code.
https://help.syncfusion.com/xamarin/pdf-viewer/working-with-custom-stamp-annotations#add-a-custom-stamp

Reply:
Feedback addressed.